airflow/models/dag.py

```diff
@@ -2372,8 +2471,13 @@ class DagModel(Base):
 
     has_task_concurrency_limits = Column(Boolean, nullable=False)
 
-    # The execution_date of the next dag run
-    next_dagrun = Column(UtcDateTime)
+    # Kept for backwards compatibility. New code should use data_interval columns.
```

Comment:
I think we will still need this -- it is the execution_date/schedule_date of the new DagRun to create -- which may be different to interval start or end.

```suggestion
```